packages/core/src/components/private/ButtonOrLink.tsx

```diff
@@ -28,6 +28,8 @@ export type Props = {
   openInNewWindow?: boolean;
   /** Rel attribute override for if the component has an href */
   rel?: string;
+  /** Add a data-tracking-name attribute. */
+  trackingName?: string;
```

Comment:
So in our repo we currently wrap button and link with another component to require a tracking name.  I was wondering if it would make sense to make this required?  I can see arguments both ways though since you aren't required to use tracking with lunar.  It just feels like for airbnb use case this is the best practice.

Reply:
I think it'd break too much to require this as there are so many consumers